app/api/public-cloud/requested-project/[id]/route.ts

```diff
@@ -12,7 +12,7 @@ const apiHandler = createApiHandler({
   validations: { pathParams: pathParamSchema },
 });
 
-export const GET = apiHandler(async ({ pathParams }) => {
```

Comment:
this route will return no data as `core service` does not support for the schema `requested project`.
let's remove this route and rename `app/api/private-cloud/request/[id]` to `app/api/private-cloud/requests/[id]` to fetch requested projects for a request.